### ISSUE TITLE
Create the About page

### ISSUE THREAD

**Issue by haltakov** [underfitted/bnomial-frontend#7]
Design and implement the first simple version of the About page.

Features of the About page:
- Explain the model of creating projects at Underfitted

If you are not sure what exactly the page should contain feel free to ask questions and discuss options in Discord! Creativity and ideas are encouraged!

**Comment by ncontrerasj** [comment]
just a doubt @haltakov, why in the bnomial's website should we explain the model of creating projects at Underfitted? maybe we should use something like [this](https://docs.underfitted.io/incubator/bnomial), or did I miss the big picture?

**Comment by haltakov** [comment]
Yes, my initial description of the issue is not very good... I think we need both.

As you say, on the About page we need to have some general information like what the project is, what are the goals we have with it, what was the motivation to build it.

At the same time, when I visit the About page of a project, I want to see who is behind it. This is what I originally meant - I think we need to tell the people that this project was built by the Underfitted community in an open manner. We let all community members contribute and we reward people with the $BNO token depending on their contributions.

What do you think?

**Comment by ncontrerasj** [comment]
Sorry for not answered back, I agree with you, in the other hand, too many things happened with the project since I choose this issue, I'm going to release it for the moment.
